lib/core/Config.cpp

```diff
@@ -73,7 +75,7 @@ json Config::get_value(const string& base_path, const string& name) {
     for (;;) {
 
         if (search_path == "" && exists("/", name)) {
-            json::json_pointer value_pointer("/" + name);
+            json::json_pointer value_pointer(fmt::format(fmt("/{:s}"), name));
```

Comment:
The motivation here was performance. I thought since fmt is supposed to be faster than simple concatenation (`+`) I might as well replace it everywhere, hoping that we would make it the standard of this project.